File: C++/operaciones.cpp
#include <iostream>

using namespace std;

int main() {
    int A, B;

    cout << "Ingrese valor: ";
    cin >> A;
    cout << "Ingrese valor: ";
    cin >> B;

    // Operaciones básicas
    int suma = A + B;
    cout << "La suma de los numeros es: " << suma << endl;

    int res = A - B;
    cout << "La resta de los numeros es: " << res << endl;

    int multi = A * B;
    cout << "La multiplicacion de los numeros es: " << multi << endl;

    float div = static_cast<float>(A) / B;
    cout << "La division de los numeros es: " << div << endl;

    // Comparaciones
    bool igual = (A == B);
    cout << "Los numeros son iguales: " << (igual ? "Si" : "No") << endl;

    if (A > B) {
        cout << "El numero mayor es: " << A << endl;
        cout << "El numero menor es: " << B << endl;
    } else if (A < B) {
        cout << "El numero mayor es: " << B << endl;
        cout << "El numero menor es: " << A << endl;
    } else {
        cout << "Ambos numeros son iguales." << endl;
    }

    return 0;
}

File: JAVA/operaciones.java
import java.util.Scanner;

public class OperacionesMatematicas {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Entrada de datos
        System.out.print("Ingrese valor: ");
        int A = scanner.nextInt();

        System.out.print("Ingrese valor: ");
        int B = scanner.nextInt();

        // Operaciones matemáticas
        int suma = A + B;
        System.out.println("La suma de los números es: " + suma);

        int res = A - B;
        System.out.println("La resta de los números es: " + res);

        int multi = A * B;
        System.out.println("La multiplicación de los números es: " + multi);

        // Manejo de la división para evitar divisiones por cero
        if (B != 0) {
            double div = (double) A / B;
            System.out.println("La división de los números es: " + div);
        } else {
            System.out.println("No se puede dividir por cero.");
        }

        // Comparaciones
        boolean igual = (A == B);
        System.out.println("¿Los números son iguales?: " + igual);

        boolean mayor = (A > B);
        System.out.println("El número menor es: " + (mayor ? B : A));
        System.out.println("El número mayor es: " + (mayor ? A : B));

        scanner.close();
    }
}
